### PR TITLE
Remove old m56 and m57 GCI releases from prow config

### PR DESCRIPTION
**Pull request by aulanov**

### CODE REVIEW

**Comment by krzyzacy** [comment]
@aulanov you'll also need to remove the entries from https://github.com/kubernetes/test-infra/blob/master/testgrid/config/config.yaml and https://github.com/kubernetes/test-infra/blob/master/jobs/config.json

File: jobs/config.json [hunk 1]
```diff
@@ -1205,34 +1205,6 @@
       "UNKNOWN"
     ]
   },
-  "ci-kubernetes-e2e-gce-gci-qa-m56": {
-    "args": [
-      "--env-file=jobs/platform/gce.env",
-      "--env-file=jobs/ci-kubernetes-e2e-gce-gci-qa-m56.env",
```
Comment (krzyzacy) [inline jobs/config.json]: also please clean up the obsoleted env files, and you shall be good.

**Comment by wonderfly** [comment]
@krzyzacy Feel free to merge if it looks good to you

**Pull request merged**